### PR TITLE
Code cleanup & Simplified LINQ expressions

### PR DESCRIPTION
**Pull request by primona**
Removed redundant code, cleaned up some of the LINQ and other minor tweaks

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ static void Main(string[] args)
             services.GetRequiredService<WalletService>();
             await services.GetRequiredService<CommandHandlingService>().InitializeAsync(services);
 
-            await _client.LoginAsync(TokenType.Bot, _config["token"]);
+            await _client.LoginAsync(TokenType.User, _config["token"]);
```
Comment (MoonMoonDogo) [inline Program.cs]: Why was this changed to ``TokenType.User``?

File: Services/RainService.cs [hunk 2]
```diff
@@ -21,25 +21,25 @@ public class RainService
...
-        private ulong _guildId;
+        private readonly ulong _guildId;
         private IGuild _guild;
         private IReadOnlyCollection<IGuildUser> _guildUsers;
 
         public RainServiceState State { get; private set; }
         public long BalanceThreshold { get; private set; }
         public TurtleWallet BotWallet { get; private set; }
-        public List<Emote> ReactionEmotes { get; private set; }
+        private List<Emote> ReactionEmotes { get; set; }
```
Comment (MoonMoonDogo) [inline Services/RainService.cs]: Minor nitpick: This should probably be a private, readonly field instead of a private readonly property.

File: Program.cs (hunk 1 above)
Comment (primona) [reply Program.cs]: I forgot to change it after testing. my bad

**Comment by primona** [comment]
These were just issues that caught my eye at first glance.

**Comment by MoonMoonDogo** [comment]
Thanks for the clarification on the token change.
I will merge this PR after [this little nitpick](https://github.com/turtlecoin/rain-turtle/pull/13/files/7e58fe5bfab8d48083166389710dfff04eba5243#r169196349) is eliminated 👍.

**Comment by MoonMoonDogo** [comment]
Thanks again for the PR!
